components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -162,6 +166,20 @@
      */
     public static final String IAT = "iat";
 
+    /***
+     * Constant for user access token expiry time.
+     */
+    public static final String USER_ACCESS_TOKEN_TIME_IN_MILLISECONDS = "userAccessTokenExpireTime";
```

Comment:
USER_ACCESS_TOKEN_TIME_IN_MILLISECONDS it will have a meaning if we define 
USER_ACCESS_TOKEN_EXP_TIME_IN_MILLISECONDS

Reply:
fixed

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/474